src/libs/ComponentSpecs.js

```diff
@@ -311,56 +290,56 @@ const prototypeSpecs = {
+  //   let foundElements = [];
+  //   const reactI13n = self._getReactI13n();
+  //   const scanTags = (self.props.scanLinks && self.props.scanLinks.tags) || DEFAULT_SCAN_TAGS;
+  //   if (!DOMNode) {
+  //     return;
+  //   }
+  //   self._subI13nComponents = [];
 
     // find all links
-    scanTags.forEach((tagName) => {
-      const collections = DOMNode.getElementsByTagName(tagName);
-      if (collections) {
-        foundElements = foundElements.concat(arrayFrom(collections));
-      }
-    });
+    // scanTags.forEach((tagName) => {
+    //   const collections = DOMNode.getElementsByTagName(tagName);
+    //   if (collections) {
+    //     foundElements = foundElements.concat([...collections]);
+    //   }
+    // });
 
     // for each link
     // 1. create a i13n node
     // 2. bind the click event
     // 3. fire created event
     // 4. (if enabled) create debug node for it
-    foundElements.forEach((element) => {
-      const I13nNodeClass = reactI13n.getI13nNodeClass() || I13nNode;
-      const i13nNode = new I13nNodeClass(self._i13nNode, {}, true, reactI13n.isViewportEnabled());
-      i13nNode.setDOMNode(element);
-      const subThis = {
-        props: {
-          href: element.href,
-          follow: true
-        },
-        getI13nNode: function getI13nNodeForScannedNode() {
-          return i13nNode;
-        }
-      };
-
-      subThis._shouldFollowLink = self._shouldFollowLink.bind(subThis);
-      subThis.executeI13nEvent = self.executeI13nEvent.bind(self);
-
-      self._subI13nComponents.push({
-        componentClickListener: listen(element, 'click', clickHandler.bind(subThis)),
-        debugDashboard: IS_DEBUG_MODE ? new DebugDashboard(i13nNode) : null,
-        domElement: element,
-        i13nNode
-      });
-      self._getReactI13n().execute('created', { i13nNode });
-    });
-  },
+  //   foundElements.forEach((element) => {
+  //     const I13nNodeClass = reactI13n.getI13nNodeClass() || I13nNode;
+  //     const i13nNode = new I13nNodeClass(self._i13nNode, {}, true, reactI13n.isViewportEnabled());
+  //     i13nNode.setDOMNode(element);
+  //     const subThis = {
+  //       props: {
+  //         href: element.href,
+  //         follow: true
+  //       },
+  //       getI13nNode: function getI13nNodeForScannedNode() {
+  //         return i13nNode;
+  //       }
+  //     };
+  //
+  //     subThis._shouldFollowLink = self._shouldFollowLink.bind(subThis);
+  //     subThis.executeI13nEvent = self.executeI13nEvent.bind(self);
+  //
+  //     self._subI13nComponents.push({
+  //       componentClickListener: listen(element, 'click', clickHandler.bind(subThis)),
+  //       debugDashboard: IS_DEBUG_MODE ? new DebugDashboard(i13nNode) : null,
+  //       domElement: element,
+  //       i13nNode
+  //     });
+  //     self._getReactI13n().execute('created', { i13nNode });
+  //   });
+  // },
 
   /**
    * _shouldFollowLink, provide a hook to check followLink.
```

Comment:
We never expose `shouldFollowLink` to any documentation, probably only use internally. This should either be a props or something else if we want to support it. (functional component won't work anyway)